html/semantics/forms/the-input-element/email-set-value.html

```diff
@@ -1,30 +1,26 @@
 <!DOCTYPE html>
+<html class="reftest-wait">
 <title>Input Email setValue</title>
 <link rel="author" href="mailto:atotic@chromium.org">
 <link rel="help" href="https://html.spec.whatwg.org/multipage/#e-mail-state-(type=email)">
 <link rel="help" href="https://crbug.com/423785">
-<script src="/resources/testharness.js"></script>
-<script src="/resources/testharnessreport.js"></script>
+<link rel="match" href="email-set-value-ref.html">
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-actions.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
 
+<input type="email">
 <input type="email">
 
 <script>
-promise_test(async () => {
-  let input = document.querySelector("input");
+(async function () {
+  let inputs = document.querySelectorAll("input");
   let unsanitized = ' foo@bar   ';
   let sanitized = unsanitized.trim();
-  await test_driver.send_keys(input, unsanitized);
-  input.select();
-  assert_equals(input.value, sanitized, "value is sanitized");
-  assert_equals(window.getSelection().toString(), unsanitized,
-    "visible value is unsanitized");
-  input.value = sanitized;
-  input.select();
-  assert_equals(window.getSelection().toString(), sanitized,
-    "visible value is sanitized after setValue(sanitized)");
-},
-"setValue(sanitizedValue) is reflected in visible text field content");
+  await test_driver.send_keys(inputs[0], unsanitized);
+  await test_driver.send_keys(inputs[1], unsanitized);
+  inputs[1].value = sanitized;
```

Comment:
```suggestion
  inputs[1].value = input[0].value;
```

I think this is better, because the original test was testing that the `.value` getter returned the sanitized value. You wouldn't need the `sanitized` variable as well.

Reply:
I don't think relying on the first input to give the correct output is a good way of testing this. I want to more explicitly specify the sanitized value.